packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -211,6 +212,10 @@ class RefreshIndicator extends StatefulWidget {
   /// Defaults to [RefreshIndicatorTriggerMode.onEdge].
   final RefreshIndicatorTriggerMode triggerMode;
 
+  /// Defines if the refresh indicator should be running.
+  /// Not running by default.
```

Comment:
Can you add more documentation and an example showing how you would use this? I am still not sure I understand why we need to add this to the RefeshIndicator widget.